modules/hmmer/hmmbuild/main.nf

```diff
@@ -0,0 +1,35 @@
+process HMMER_HMMBUILD {
+    tag '$meta.id'
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hmmer=3.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.3.2--h87f3376_2':
+        'quay.io/biocontainers/hmmer:3.3.2--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(alignment)
+
+    output:
+    tuple val(meta), path("*.hmm"), emit: hmm
```

Comment:
`.hmm` file should be gzipped: https://github.com/nf-core/modules/blob/11cf45043cf61ab88fa02e2f67d83308ca724fe2/modules/hmmer/hmmalign/main.nf#L28 [^1]

IIRC when making the two other HMM modules, (most) HMMer tools supported gzipped input. You can ignore this message if build is an exception.

[^1]: https://nf-co.re/docs/contributing/modules#general

Reply:
Build can take a gzipped file, and autodetects. I have however never seen gzipped hmm files, probably because they're mostly not so large.

Reply:
I also generatlly don't see them too, but in other cases you may end up generating hundreds of HMM files (e.g. annotating a genome) so it is preferred -definitely as the tool accepts gzipped files.

Reply:
Done.